### ISSUE TITLE
Change sidebar to be lighter color than chat window

### ISSUE THREAD

**Issue by tojaroslaw** [LanikSJ/slack-dark-mode#134]
**Is your feature request related to a problem? Please describe.**
The sidebar and chat window are the same color.  In the picture you provide in the readme, they are not and it looks much better that way as it is far easier to distinguish between fields

**Describe the solution you'd like**
Make the sidebar a different color from the chat

**Describe alternatives you've considered**
manually altering the CSS file by looking at older versions and putting those changes back in

**Additional context**
see the screenshot in the readme

**Comment by mikepenz** [comment]
@tojaroslaw I am not sure if this is expected. 

But as the README noes you can define your own custom colors via:

```
You can tweak your own css colors by creating a new file called custom-dark-theme.css,
which will be used to overwrite the current theme.
```

For example I have a `custom-dark-theme.css` file with the following content:
```css
:root {
    /* General background */
    --main-bg-color: #282A36;
    /* Text input, dividers */
    --main-dark-highlight: #141622;
    /* Message bar top (new messages), Currently selected chat, Nes messages line */
    --main-highlight: #151724;
    /* Main text in the chat windows */
    --main-text: #e6e6e6;
    /* Secondary text (70% of main text) */
    --secondary-text: #e6e6e6B3;
    /* Sidebar background color */
    --sidebar-bg-color: #1f212b;
    /* Thread background color */
    --thread-bg-color: #1f212b;
    /* Based on the usage should be link hover color */
    --text-hover: #c7c7c7;
    /* General hightlighs and warnings (important and new stuff) */
    --highlight-and-warnings: #bf360c;
    /* Light hightlighs and warnings */
    --highlight-and-warnings-light: #ef440f;
    /* General code text */
    --main-code: #FF9800;
  }
```

**Comment by krazyito65** [comment]
Yea, I created that custom file for exactly this purpose.  Everybody has their own preferences for what they think something should look like, so the maintainer of the code base shouldn't have to be heckled all the time about what other people think it should be.  

So I provided a file where people can overwrite specific elements with their own tweaks, as Mike as noted.

**Comment by krazyito65** [comment]
this is also related to #103

**Comment by krazyito65** [comment]
I think this can be closed becasue of #103
